test/IDE/complete_pound_keypath.swift

```diff
@@ -31,7 +31,7 @@ class ObjCClass : NSObject {
 }
 
 func completeInKeyPath2() {
-  _ = #keyPath(ObjCClass.#^IN_KEYPATH_2^#
+  _ = #keyPath(ObjCClass.p#^IN_KEYPATH_2^#
```

Comment:
These changes are for the sake of making tests pass, but it seems that enabling this feature makes code completion stop suggesting the properties in this specific case unless you help it with `p`. I don't think this was supposed to happen. Any idea why?